index.html

```diff
@@ -1646,29 +1646,38 @@ <h3>Issuer</h3>
       </section>
 
       <section>
-        <h3>Issuance Date</h3>
+        <h3>Validity Period</h3>
 
         <p>
-This specification defines the <code>issuanceDate</code> <a>property</a> for
-expressing the date and time when a <a>credential</a> becomes valid.
+This specification defines the <code>validFrom</code> <a>property</a> to help an
+issuer to express the date and time when a <a>credential</a> becomes valid and
+the <code>validUntil</code> <a>property</a> for expressing the date and time
+when a <a>credential</a> ceases to be valid.
         </p>
 
         <dl>
-          <dt><var>issuanceDate</var></dt>
+          <dt><var>validFrom</var></dt>
           <dd>
-A <a>credential</a> MUST have an <code>issuanceDate</code> <a>property</a>. The
-value of the <code>issuanceDate</code> <a>property</a> MUST be a string value of
+A <a>credential</a> MUST have an <code>validFrom</code> <a>property</a>. The
+value of the <code>validFrom</code> <a>property</a> MUST be a string value of
 an [<a data-cite="XMLSCHEMA11-2#dateTime">XMLSCHEMA11-2</a>] combined
 <code>date-time</code> string representing the date and time the
 <a>credential</a> becomes valid, which could be a date and time in the future.
 Note that this value represents the earliest point in time
 at which the information associated with the <code>credentialSubject</code>
 <a>property</a> becomes valid.
           </dd>
+          <dt><var>validUntil</var></dt>
+          <dd>
+  If present, the value of the <code>validUntil</code> <a>property</a> MUST be
+  a string value of an [<a data-cite="XMLSCHEMA11-2#dateTime">XMLSCHEMA11-2</a>]
+  <code>date-time</code> representing the date and time the <a>credential</a>
+  ceases to be valid.
```

Comment:
to match the description/definition of `validFrom`
```suggestion
  combined <code>date-time</code> string representing the date and time the
  <a>credential</a> ceases to be valid, which could be a date and time in the past.
  Note that this value represents the latest point in time at which the information
  associated with the <code>credentialSubject</code> <a>property</a> is valid.
```